### PR TITLE
Without Count

### PR DESCRIPTION
**Pull request by victornamuso**
Inspired by this PR (https://github.com/stefanpenner/ember-inflector/pull/98) and my own need. 

Updated so a third parameter would be specified for wordOnly leaving the current default behavior intact. Also updated tests so the change is passing. Finally updated the README for examples of using the template helpers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,28 @@ pluralize("taco"); // tacos
...
+
+Specify a count with the word, with the pluralization being based on the number of items.
+```helpers
+{{pluralize 1 "taco"}} //1 taco
+{{pluralize 2 "taco"}} //2 tacos
+```
+
+Specify a count with the word, with the pluralization being based on the number of items. Specify true for the last parameter to return on the word without the number.
+```helpers
+{{pluralize 1 "taco" true}} //taco
```
Comment (et) [inline README.md]: How about:

```handlebars
{{pluralize 1 "taco" wordOnly=true}}
```

And actually `withoutCount` might be a better name for the option.

Comment (victornamuso) [inline README.md]: I like this:
```
{{pluralize 1 "taco" withoutCount=true}}
```
I'll update the code and resubmit.

**Comment by victornamuso** [comment]
@et Any idea why the CI is failing? Locally it's working and it's passing on my fork: https://travis-ci.org/parablesoft/ember-inflector

File: tests/unit/integration-test.js [hunk 2]
```diff
@@ -56,37 +56,97 @@ test("helpers - pluralize - bound count 1", function(assert) {
   assert.equal(this.$().text(), '1 orange');
 });
 
+test("helpers - pluralize - bound count 1, without count", function(assert) {
+  this.set('count', 1);
+  this.set('singular', 'orange');
+	this.set('withoutCount', true);
```
Comment (et) [inline tests/unit/integration-test.js]: indentation

**Comment by victornamuso** [comment]
Thanks @et. Looks like we are good.

File: README.md [hunk 3]
```diff
@@ -27,3 +27,28 @@ pluralize("taco"); // tacos
...
+
+Specify a count with the word, with the pluralization being based on the number of items.
+```helpers
+{{pluralize 1 "taco"}} //1 taco
+{{pluralize 2 "taco"}} //2 tacos
+```
+
+Specify a count with the word, with the pluralization being based on the number of items. Specify `withoutCount=true` to return on the word without the number.
+```helpers
+{{pluralize 1 "taco" withoutCount=true}} //taco
```
Comment (stefanpenner) [inline README.md]: @rwjblue should this be dasherized?

Comment (victornamuso) [inline README.md]: @stefanpenner That's pretty consistent with conventions these days isn't it?

Comment (stefanpenner) [inline README.md]: ya, just spoke to @rwjblue he agrees.

Comment (victornamuso) [inline README.md]: Cool, I'll make the changes and re-submit, probably at some point tomorrow.

Comment (stefanpenner) [inline README.md]: bikeshedding here, should it maybe be `count=true` (Default)  `count=false` ? the with may not be required at all.

Comment (victornamuso) [inline README.md]: Even better

**Comment by stefanpenner** [comment]
@victornamuso sorry for letting this one sit. I easily loose active PR in GH's UI.

File: README.md (hunk 3 above)
Comment (victornamuso) [inline README.md]: @stefanpenner Internally the variable count is used and the term count is used in the README and in the comments. Back to without-count?

Comment (victornamuso) [inline README.md]: @stefanpenner Bumping. Just wanted to make sure you saw my last comment.

Comment (victornamuso) [inline README.md]: @stefanpenner Hey man, just wanted to bump again. Let me know what you'd like done with it.

**Comment by RuslanZavacky** [comment]
@stefanpenner @victornamuso guys, can I poke you a bit about this PR? Maybe we can move it forward? :)

**Comment by victornamuso** [comment]
@RuslanZavacky I'll just change it up to without-count and re-submit. Hopefully will get this done at the end of the week.

**Comment by stefanpenner** [comment]
Ya looks like the withoutCount needs to still be dasherized. Once that happens we can move forward

**Comment by victornamuso** [comment]
@stefanpenner Yes sir. That's the change that I'm going to make. Will pull in the latest, make the adjustments, then re-submit.

**Comment by victornamuso** [comment]
@stefanpenner Should be all set. Let me know if anything further is needed.

**Comment by stefanpenner** [comment]
released as v1.12.0 🎉

**Comment by victornamuso** [comment]
@stefanpenner One last thing. Seems like #98 can be closed now as well.

**Pull request merged**